tests/regression/57-affeq/14-norm_inv.c

```diff
@@ -0,0 +1,33 @@
+//SKIP PARAM: --set ana.activated[+] affeq  --set ana.affeq.matrix "list" --sem.int.signed_overflow "assume_none"  --enable ana.int.interval
+// Snippet from https://github.com/sosy-lab/sv-benchmarks/blob/master/c/nla-digbench/hard2.c
+// Normalization should be triggered when an invertible expression is assigned.
```

Comment:
It may have been a fixpoint issue or crash at some point. Can we add asserts that make sense here?

Reply:
That would be great. I added some to 57-affeq/01-rel_simple from Karr's paper, but I'm not familiar enough with the details to know what should be asserted regarding this description.

Reply:
I was not able to dig exactly into what it was (the commit that added this was improperly rebased and has some unresolved conflicts now), but suspect a fixpoint issue.

I would propose we just add a comment that says this was likely a fixpoint issue and still leave it here: if it caused a subtle bug we minimized from sv-comp tasks once, a similar bug may resurface later so I think it would be unwise to remove it, even if we don't know what exactly was the issue.